lib/ebaze/accounts/user.ex

```diff
@@ -0,0 +1,18 @@
+defmodule Ebaze.Accounts.User do
+  use Ecto.Schema
+  import Ecto.Changeset
+
+  schema "users" do
+    field :password, :string
+    field :username, :string
```

Comment:
Length requirements for password and uniqueness for username?

Reply:
e91e6a2172fb6cbae80310a4d34a79efd5b4bb65